frontend/src/model/beta/registry/CodeLine.tsx

```diff
@@ -29,14 +29,14 @@ export default function CodeLine({ line, icon = <CodeIcon /> }: CodeLineProps) {
           handleButtonClick()
         }}
         onKeyDown={(e) => {
-          if (e.key === 'Enter' || e.key === 'Spacebar') {
+          if (e.key === 'Enter' || e.key === 'Spacebar' || e.key === ' ') {
             handleButtonClick()
           }
         }}
       >
         <Stack direction='row' spacing={2} alignItems='center'>
           {icon}
-          <Typography>{`${line}`}</Typography>
+          <Typography>{line}</Typography>
```

Comment:
I think the $ was intended to make it stand out as a line of code (like a terminal), do we not want to keep it?